packages/block-editor/src/components/border-radius-control/test/utils.js

```diff
@@ -121,26 +121,26 @@ describe( 'getAllValue', () => {
 			expect( getAllValue( values ) ).toBe( '1em' );
 		} );
 
-		it( 'should fall back to null when values are undefined', () => {
+		it( 'should fall back to undefined when values are undefined', () => {
 			const values = {
 				bottomLeft: undefined,
 				bottomRight: undefined,
 				topLeft: undefined,
 				topRight: undefined,
 			};
-			expect( getAllValue( values ) ).toBe( null );
+			expect( getAllValue( values ) ).toBe( undefined );
 		} );
 	} );
 
 	describe( 'when provided invalid values', () => {
 		it( 'should return px when passed an array', () => {
-			expect( getAllValue( [] ) ).toBe( null );
+			expect( getAllValue( [] ) ).toBe( undefined );
 		} );
 		it( 'should return px when passed a boolean', () => {
-			expect( getAllValue( false ) ).toBe( null );
+			expect( getAllValue( false ) ).toBe( undefined );
 		} );
 		it( 'should return px when passed undefined', () => {
-			expect( getAllValue( false ) ).toBe( null );
+			expect( getAllValue( false ) ).toBe( undefined );
 		} );
```

Comment:
I should probably push a couple of updates to these tests:

- the test names seem wrong (probably copy/paste)
- the last test should be probably changed to use `undefined` instead of `false`